packages/ui-kit/src/design-system.ts

```diff
@@ -121,6 +123,8 @@ export const colors: ThemeColors = {
     link: customProperties.colorPrimary25,
     linkHover: customProperties.colorPrimary,
     linkNavigation: customProperties.colorInfo,
+    // Headlines
+    headlinePrimary: '#003037',
```

Comment:
Sorry to be picky but can't we use one of our existing Design System colors, like `colorAccent30`? Why introducing a new one?

cc @Shecki 

This is how it looks (left side) compared to this new color `#003037` (right side).

<img width="1840" alt="image" src="https://user-images.githubusercontent.com/1110551/162761924-2415ed24-7f5c-4195-9cb1-fab66f977352.png">

Reply:
Thanks for the clarification Sven.  We'll have to talk about the direction of the Docs UI Kit and design system sometime. It's currently derived from the MC UI Kit but like in this example it seems that corporate design is moving independently (but there is no design system or UI Kit implementation available for corporate elements).  

For today good enough this way.